crm_lead_vat/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "VAT in leads",
+    "summary": "Add VAT field to leads",
+    "version": "12.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "http://www.antiun.com",
```

Comment:
please change the website to: https://github.com/OCA/crm

Reply:
Thanks @tbaden change done.